src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -45,6 +50,23 @@ Results<Key> getKeys() throws MeilisearchException {
         return result;
     }
 
+    /**
+     * Retrieves keys from the client
+     *
+     * @param param accept by the keys route
+     * @return List of key instance
+     * @throws MeilisearchException if client request causes an error
+     */
+    Results<Key> getKeys(KeysQuery param) throws MeilisearchException {
+        URLBuilder urlb = new URLBuilder();
+        urlb.addSubroute("keys")
```

Comment:
I left a similar comment in the documents PR, I think this should be done inside of the `KeysQuery`.